src/graph/dinic.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "src/cpp-template/header/rep.hpp"
+#include "src/cpp-template/header/type-alias.hpp"
+
+#include <algorithm>
+#include <cassert>
+#include <limits>
+#include <queue>
+#include <vector>
+
+namespace luz {
+
+  template < typename cap_type >
+  class Dinic {
```

Comment:
```suggestion
  class MaxFlowGraph {
```

ac-library とかに合わせてこんなかんじにしたほうがいい気がしてきたけどどうですか？

これ変更するならファイル名も `dinic.hpp` にするよりは `max-flow.hpp` とかにしたほうがいいかも